assets/scss/templates/_guidelines.scss

```diff
@@ -1,12 +1,12 @@
-main {
-  padding: 0 5%;
-}
+// main {
+//    padding: 0 5%;
+// }
 
-section {
+.section-module {
   width: 100%;
   max-width: 1366px;
   margin: 0 auto;
-  padding: 80px 0;
+  padding: 80px 5%;
   border-bottom: 1px solid #CCCCCC;
```

Comment:
not use variables for colors?

Reply:
Si vamos a utilizar variables, eso solo era una prueba. Lo puedo eliminar.